bot/cogs/utils.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from email.parser import HeaderParser
+from io import StringIO
+
+
+from discord import Embed
+from discord.ext.commands import AutoShardedBot, Context, command
+
+from bot.constants import Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+
+class Utils:
+    """
+    Helpful commands
+    """
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+
+        self.base_pep_url = "http://www.python.org/dev/peps/pep-"
+        self.base_github_pep_url = "https://raw.githubusercontent.com/python/peps/master/pep-"
+
+    @command(name="pep()", aliases=["pep", "get_pep"])
+    @with_role(Roles.verified)
+    async def pep_search(self, ctx: Context, pep_number: str):
+        """
+        Fetches information about a PEP and sends it to the user
+        """
```

Comment:
multiline docstrings should be followed by blank line.

Reply:
i think you should punctuate your docstring, by the way.